fabric_plugin/tasks.py

```diff
@@ -23,9 +23,18 @@
 from contextlib import contextmanager
 
 import requests
-from fabric import Connection, task
+from fabric2 import Connection, task
 from invoke import Task
-from paramiko import RSAKey, ECDSAKey, Ed25519Key, SSHException
+from paramiko import RSAKey, ECDSAKey, SSHException
+
+''' This is done because on 5.0.5 manager and older we will have
+ 1.X paramiko version'''
+try:
+    from paramiko import Ed25519Key
+
+    PARAMIKO_VERSION = 2
+except ImportError:
+    PARAMIKO_VERSION = 1
```

Comment:
```suggestion
try:
    from paramiko import Ed25519Key
    ED25519_AVAILABLE = True
except ImportError:
    ED25519_AVAILABLE = False
```
i suggest naming this about feature availability and not about versions, because what if paramiko upgrades to 3 and then to 4?
with this name, it suggests that we only care if ed25519 is available and not what version is it

Reply:
done